x-pack/docs/en/security/securing-communications/node-certificates.asciidoc

```diff
@@ -76,19 +83,20 @@ signing requests (CSR) for the nodes in your cluster. For more information, see
 --
 
 . Copy the node certificate to the appropriate locations.
+
+.. Create a folder in the configuration directory on each {es} node to contain
+your security certificates. For example, create a `certs` folder in the
+`/home/es/config/certs` directory.
 +
---
-Copy the applicable `.p12` file into a directory within the {es} configuration
-directory on each node. For example, `/home/es/config/certs`. There is no need
-to copy the CA file to this directory.
+NOTE: The {ref}/settings.html#config-files-location[{es} configuration
+directory] varies depending on your {es} installation.
 
-For each additional Elastic product that you want to configure, copy the
-certificates to the relevant configuration directory. 
---
+.. Copy the node certificates into the `certs` directory that you created in the
+previous step.
 
-NOTE: If you choose not to use `elasticsearch-certutil`, the certificates that
-you obtain must allow for both `clientAuth` and `serverAuth` if the extended key
-usage extension is present. The certificates need to be in PEM or PKCS#12
-format. Although not required, it is highly recommended that the certificate
-contain the DNS names and/or IP addresses of the node so that hostname
-verification can be used.
+.. Copy the `.p12` keystore into the {es} configuration directory. If this file
+does not exist in the location that {es} expects, permission checks will fail
```

Comment:
> permission checks will fail

Should this be added to the list of bootstrap checks in https://www.elastic.co/guide/en/elasticsearch/reference/master/bootstrap-checks-xpack.html?

Reply:
@lcawl, in theory I agree. However, I don't know that Elasticsearch completes this validation as a bootstrap check. I believe the behavior is that permission checks will fail, but there's no validation to check whether the keystore file is located in the Elasticsearch configuration directory. 

@tvernum, can you please confirm this behavior?

Reply:
It's not a bootstrap check, so I don't think we should add it to that doc.

Rather than `permission checks` which somehow manages to be both overly specific and strangely vague at the same time could we say
 ```suggestion
is placed outside of this directory, {es} will not be permitted to read the file and will fail to start. 
```

Reply:
@tvernum, thanks for the input on that check. I agree that we can refine this statement. I'm 👍 on your revision, but wanted to propose a slight tweak. Please let me know what you think.

>Copy the `.p12` keystore file into the {es} configuration directory. Elasticsearch will fail to start if the keystore file is located anywhere except this directory.

Reply:
Your revision LGTM.